loader:
  batch_size: 8
  shuffle: true
  num_workers: 0

train:
  use_cuda: true
  model: "alexnet"
  save_model_name: "best3"
  epochs: 10
  learning_rate: 0.001
  momentum: 0.09
